andela/persephone-ah-backend
# Ch setup swagger docs 166816096

#### What does this PR do?
setup swagger documentation

#### Description of Task to be completed?
> - Add swagger.json file
> - Add new endpoint api-docs

#### How should this be manually tested?
> - Clone this branch
> - Checkout to this branch
> - Run npm start to kick start the server
> - Then open this link on your browser: [local-docs](http://localhost:3000/api-docs)
> - You should be able to view the documentation now

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
Pt story link [166816096](https://www.pivotaltracker.com/story/show/166816096)

#### Screenshots (if appropriate)
NA

#### Questions:

NA
